jose/jwt.py

```diff
@@ -184,6 +184,14 @@ def get_unverified_claims(token):
     except:
         raise JWTError('Error decoding token claims.')
 
+    try:
+        claims = json.loads(claims.decode('utf-8'))
```

Comment:
Are you ok with reassigning the variable here? I felt it was the least intrusive fix.